pkg/api/v1/routes/handlers_test.go

```diff
@@ -423,7 +423,7 @@ func TestDeleteServer(t *testing.T) {
 					Times(1)
 			},
 			func(t *testing.T) *http.Request {
-				url := fmt.Sprintf("/api/v1/serverDelete/%v", mockServerID)
+				url := fmt.Sprintf("/api/v1/server/%v", mockServerID)
```

Comment:
The target url should be `api/v1/servers/{serverID}`. Note `servers` plural, not `server` singular.